install_scripts/templates/kubernetes/init.sh

```diff
@@ -123,7 +124,7 @@ EOF
 
 initKube() {
     logStep "Verify Kubelet"
-    if ! ps aux | grep -qE "[k]ubelet"; then
+    if [ ! -e "/etc/kubernetes/manifests/kube-apiserver.yaml" ]; then
```

Comment:
This should be a more reliable test of whether kubeadm has initialized since the kubelet process is started separately with systemctl.